src/pages/writing-nft/index.vue

```diff
@@ -61,6 +61,26 @@
           </li>
         </ul>
       </nav>
+      <div
+        v-if="currentTab === 'trending'"
+        class="flex mt-[24px] invisible desktop:visible"
```

Comment:
- Use hidden instead of invisible at mobile?
- Use `grid` or `flex`? If flex, please add `flex-wrap` (Personally, I think grid layout is better)

Reply:
turns out grid is actually simpler esp with tailwind